docs/azure_jumpstart_arcbox/ITPro/_index.md

```diff
@@ -321,7 +321,7 @@ ArcBox uses an advanced automation flow to deploy and configure all necessary re
 
   ```shell
   terraform init
-  terraform plan -out=infra.out
+  terraform plan -out infra.out
```

Comment:
It should work in Bash https://developer.hashicorp.com/terraform/cli/commands/plan#planning-options
![image](https://user-images.githubusercontent.com/96136892/235188341-9891ad7b-4045-4f71-815f-e59e45e32811.png)